packages/telemed-ehr/app/src/pages/EditEmployee.tsx

```diff
@@ -16,6 +16,7 @@ export default function EditEmployeePage(): JSX.Element {
   const navigate = useNavigate();
   const [isActive, setIsActive] = useState<boolean>();
   const [user, setUser] = useState<User>();
+  const [userName, setUserName] = useState<string | undefined>(undefined);
```

Comment:
I think having separate state for userName is unnecessary since we already have a state for user and userName is derived from it.

Reply:
Was thinking that we should have a variable because `userName` is used in multiple spots, and because we have to wait for the user profile to load before setting it, it should be a state variable. The definition is kind of long:
`"${(zapEHRUser as User).profileResource?.name?.[0].given?.[0]} ${(zapEHRUser as User).profileResource?.name?.[0].family}"`